docs/mkdocs.yml

```diff
@@ -169,15 +169,14 @@ pages:
 - ['terms/filesystem.md', '**HIDDEN**']
 - ['terms/image.md', '**HIDDEN**']
 
-
-# Project:
-- ['project/index.md', '**HIDDEN**']
-- ['project/who-written-for.md', 'Project', 'README first']
-- ['project/set-up-prereqs.md', 'Project', 'Set up prerequisites'] 
-- ['project/set-up-dev-env.md', 'Project', 'Work with a development container'] 
-- ['project/test-and-docs.md', 'Project', 'Run tests and test documentation']
-- ['project/make-a-contribution.md', 'Project', 'Make a project contribution'] 
-- ['project/advanced-contributing.md', 'Project', 'Advanced contributing']
-- ['project/get-help.md', 'Project', 'Where to get help']
-- ['project/coding-style.md', 'Project', 'Coding style guide']
-- ['project/doc-style.md', 'Project', 'Documentation style guide']
+# Birthday Tree
+- ['helloworld/who-written-for.md', 'Project', 'README first for Contributors']
```

Comment:
@moxiegirl LOLz, yes. I know it's hectic, so slip ups will happen, trying to give extra attention :)

Also (I pinged you there, as well); might wanna watch this; https://github.com/docker/docker/pull/11373#issuecomment-79312647 because that will need to be published to docs as soon as it's merged